doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+        joint_model_group_pose: controllers_initial_pose_
+    # Settings for ros_control control loop
+    generic_hw_control_loop:
+        loop_hz: 300
+        cycle_time_error_threshold: 0.01
+    # Settings for ros_control hardware interface
+    hardware_interface:
+        joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+            - panda_finger_joint1
+        sim_control_mode: 1  # 0: position, 1: velocity
+    # Publish all joint states
+    # Creates the /joint_states topic necessary in ROS
+    joint_state_controller:
+        type: joint_state_controller/JointStateController
+        publish_rate: 50
+    panda_arm_controller:
+        type: effort_controllers/JointTrajectoryController
+        joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+        gains:
+            panda_joint1: { p: 12000, d: 50, i: 0.0, i_clamp: 10000 }
+            panda_joint2: { p: 30000, d: 100, i: 0.02, i_clamp: 10000 }
+            panda_joint3: { p: 18000, d: 50, i: 0.01, i_clamp: 1 }
+            panda_joint4: { p: 18000, d: 70, i: 0.01, i_clamp: 10000 }
+            panda_joint5: { p: 12000, d: 70, i: 0.01, i_clamp: 1 }
+            panda_joint6: { p: 7000, d: 50, i: 0.01, i_clamp: 1 }
+            panda_joint7: { p: 2000, d: 20, i: 0.0, i_clamp: 1 }
+
+        constraints:
+            goal_time: 2.0
+        state_publish_rate: 25
+
+    panda_hand_controller:
+        type: effort_controllers/JointTrajectoryController
+        joints:
+            - panda_finger_joint1
+            - panda_finger_joint2
+
+        gains:
+            panda_finger_joint1: { p: 5, d: 3.0, i: 0, i_clamp: 1 }
+            panda_finger_joint2: { p: 5, d: 1.0, i: 0, i_clamp: 1 }
+
+        state_publish_rate: 25
+
+    controller_list:
+        - name: panda_arm_controller
+            action_ns: follow_joint_trajectory
+            type: FollowJointTrajectory
+            default: true
+            joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+        - name: panda_hand_controller
+            action_ns: follow_joint_trajectory
+            type: FollowJointTrajectory
+            default: true
+            joints:
+            - panda_finger_joint1
+            - panda_finger_joint2
+
+
+7. Adjust auto-generated ros_controllers.launch
+-----------------------------------------------
+
+Fill the :code:`args` in line 9 with:
+
+.. code-block:: xml
+
+    joint_state_controller panda_hand_controller panda_arm_controller
+
+-------------------------------
```

Comment:
I had to modify the _gazebo.launch_ to load the xacro file. The default launch file generated from MoveIt Setup Assistant was trying to load the xacro as urdf. Maybe it was mandatory to generate the URDF from the Setup Assistant? 
![Screenshot from 2020-07-16 16-22-23](https://user-images.githubusercontent.com/18074378/87682819-98b59600-c780-11ea-9de6-f232e71a8b45.png)

Reply:
We don't touch to the URDF in the setup assistant tutorial. In that tutorial, we create SRDF files that are used particularly by MoveIt.

You are right though, the auto-generated file has the line:
`<param name="robot_description" textfile="$(arg urdf_path)" /`  to load the robot description to parameter server. I forgot to mention this. Thanks for pointing out!